components/shared/blocksRenderer.tsx

```diff
@@ -1,31 +1,75 @@
-import { HeroBlock } from "./ExampleHero";
+import FeatureBlocks, { FeatureItem } from "./Blocks/Features";
+import FAQ from "./Blocks/FAQ";
+import FeatureHorizontalCarousel from "./Blocks/FeatureCarousel";
+import Pricing from "./Blocks/Pricing";
+import Banner from "./Blocks/Banner";
 
-// Define the expected shape of the block
 interface Block {
   __typename: string;
   title?: string | null;
-  // Add any other properties your block may have
+  description?: string | null;
+  allPlans?: { title: string | null }[] | null;
+  plans?:
+    | {
+        planTier: string | null;
+        planDescription: string | null;
+        price: string | null;
+        subPriceText: string | null;
+        actions: {
+          label: string | null;
+          url: string | null;
+          variant?: string | null;
+          size?: string | null;
+        } | null;
+      }[]
```

Comment:
Should this be it's own interface/template?

Reply:
extrapolated